### DISCUSSION TITLE
No audio produced from connected Shairport

### DISCUSSION THREAD

**Discussion by pauljohn2000** [mikebrady/shairport-sync#1729] [Q&A]
I've started shairport on a Raspi 3 with an IQaudIODAC Pro card with:
**sudo service shairport-sync start** 

The service allows connection on my Mac but no sounds are produced.  Only when shairport-sync gets launched directly from the command line does it work.  Is there something I missed in the config or setup to get this working right?

**Comment by mikebrady** [comment]
Thanks for the question, but this is really best dealt with by [opening an issue](https://github.com/mikebrady/shairport-sync/issues/new/choose). Meantime, would [this](https://github.com/mikebrady/shairport-sync/blob/master/ADVANCED%20TOPICS/PulseAudioAndPipeWire.md) be relevant?
